app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsViewModel.kt

```diff
@@ -122,9 +129,9 @@ class GroupConversationDetailsViewModel @Inject constructor(
         }
     }
 
+    //TODO: some duplication here with ConversationListViewModel
```

Comment:
so what is `TODO` here, what should be done? Remove that duplication?

Reply:
no idea yet :P just adde here TODO to keep in mind to indeed remove that duplication